client/checkout/blocks/confirm-upe-payment.js

```diff
@@ -22,6 +27,8 @@ export default async function confirmUPEPayment(
 	const name =
 		`${ billingData.first_name } ${ billingData.last_name }`.trim() || '-';
 
+	const selectedUPEPaymentType = api.getSelectedUPEPaymentType();
```

Comment:
I'd prefer to move this right before `if ( PAYMENT_METHOD_IDS.AFTERPAY_CLEARPAY === selectedUPEPaymentType ) {`.

Reply:
This line is now gone since we are passing `selectedUPEPaymentType` as a param from the `upe-fields` and `upe-split-fields` function.

Reply:
Changes in this commit - 242cb56